src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -84,6 +84,11 @@
   -->
   <Import Project="$(MSBuildThisFileDirectory)versioning.targets" Condition="'$(ExcludeVersioningImport)'!='true'" />
 
+  <!--
+    Import the localization target
+  -->
+  <Import Project="$(MSBuildThisFileDirectory)localization.targets" Condition="'$(IncludeLocalizationImport)'=='true'" />
+
```

Comment:
Right now it looks like this imports is opt-in, but should we use some other trigger to determine whether or not to import? Perhaps the existents of some xlf file?